packages/builders/src/components/Assertions.ts

```diff
@@ -41,7 +45,7 @@ export const jsonOptionValidator = s
 		label: labelValueDescriptionValidator,
 		value: labelValueDescriptionValidator,
 		description: labelValueDescriptionValidator.optional,
-		emoji: emojiValidator.optional,
+		emoji: jsonEmojiValidator.optional,
```

Comment:
```suggestion
		emoji: emojiValidator,
```
emojiValidator is `nullish`

Reply:
@imranbarbhuiya did you see this?